packages/cli/lib/lib/webpack/webpack-base-config.js

```diff
@@ -84,6 +85,14 @@ module.exports = function(env) {
 
 	let tsconfig = resolveTsconfig(cwd, isProd);
 
+	let postcssPlugins;
+
+	try {
+		postcssPlugins = loadPostcssConfig.sync(cwd).plugins;
```

Comment:
sounds good to me, just wondering if we should load more than just the plugins?

Reply:
Awesome!

Okay, forgive the long response, but here are my thoughts on loading more than just the plugins from a PostCSS configuration.

The [other options](https://github.com/michael-ciniawsky/postcss-load-config#options) are `to`, `map`, `from`, `parser`, `syntax`, and `stringifier`.

I am confident that `to` and `from` can be ignored because webpack determines those locations. I am somewhat confident that `parser`, `syntax`, and `stringifier` can be ignored at this time, because `css-loader` and `style-loader` can still choke on non-CSS syntaxes.

I am on the fence about `map`. On the one hand, it can be ignored, because source map configuration needs to be unified across all CSS loaders in order to be useful. However, if PostCSS configuration enables or disables CSS source maps for a project, then that does sound useful. The question is whether PostCSS configuration is the most logical place to put control over CSS source maps.